netcdf_output.py

```diff
@@ -1103,7 +1103,7 @@ def netCDF_packaging(VX, VY, DX, DY, INTERPMASK, CHIPSIZEX, CHIPSIZEY, SSM, SSM1
         var.setncattr('dr_to_vr_factor_description', 'multiplicative factor that converts slant range '
                                                      'pixel displacement dr to slant range velocity vr')
 
-        M11 = offset2vy_2 / (offset2vx_1 * offset2vy_2 - offset2vx_2 * offset2vy_1)
+        M11 = offset2vy_2 / (offset2vx_1 * offset2vy_2 - offset2vx_2 * offset2vy_1) / scale_factor_1
```

Comment:
@jhkennedy Looking at Yang's Word document that I used for the granule correction, it is not trivial to me how autoRIFT should be corrected to be in agreement with that document. May be a safer bet to ask @leiyangleon to review it?

Reply:
These do appear to match directly their counterparts in the correction script:
https://github.com/nasa-jpl/its_live_production/blob/main/src/tools/correct_v2_s1_granules.py#L257